contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.6.0--2.7.0.sql

```diff
@@ -0,0 +1,10 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
```

Comment:
Can you kindly make sure whether do we want to have one such upgrade script for babelfishpg_common extension as well?

Reply:
We can bump it up later when we need any SQL changes in the common extension